src/extensions/query-translator/clients/const.ts

```diff
@@ -0,0 +1,104 @@
+export const nodePropsQuery = `CALL apoc.meta.data()
+YIELD label, other, elementType, type, property
+WHERE NOT type = "RELATIONSHIP" AND elementType = "node"
+WITH label AS nodeLabels, collect(property) AS properties
+RETURN {labels: nodeLabels, properties: properties} AS output
+`;
+
+export const relPropsQuery = `
+CALL apoc.meta.data()
```

Comment:
we should add in the extension description (in ExtensionModal) that APOC is a prereq. for using the extension. it's theoretically possible people try the extension without APOC core.

Reply:
Where we want to add it?